addons/web_tour/static/src/tour_service/tour_utils.js

```diff
@@ -416,6 +427,41 @@ export class RunningTourActionHelper {
         return this.anchor;
     }
 
+    /**
+     * **blur** action by default for input cause troubles mainly when input is in dialog.
+     * That's why we prefer **tab** action for inputs.
+     * @param {Node} node
+     * @returns {"blur"|"enter"|"tab"|null}
+     */
+    _get_edit_confirm_action = (node) => {
+        const tagNode = getTag(node, true);
+        const confirmActions = {
+            input: [
+                ["class", "o_searchview_input", null],
+                ["parentClass", ".o_list_view", "click .o_selected_row"],
```

Comment:
```suggestion
                ["parentClass", "o_list_view", "click .o_selected_row"],
```
?

Reply:
can't we blur (or null) for editable lists?

Reply:
![image](https://github.com/odoo/odoo/assets/127866627/cd2df54b-d3d8-4b5c-b889-a0d365d38efa)

Reply:
Of course, but `parentClass`, so you know it's a class... for `o_searchview_input`, you add the dot yourself